app/views/sufia/admin/_sidebar.html.erb

```diff
@@ -21,7 +21,11 @@
                     </li>
                     <li><%= link_to curation_concerns.admin_workflow_path do %>
                       <span class="fa fa-code-fork"></span> <%= t('sufia.admin.sidebar.workflow') %>
-                        <% end %>
+                      <% end %>
```

Comment:
The indent changed here such that it no longer matches up with the `link_to` call.  Can you fix it?